lib/graphql_devise/mount_method/operation_preparer.rb

```diff
@@ -0,0 +1,42 @@
+require_relative 'operation_preparers/default_operation_creator'
+require_relative 'operation_preparers/mutation_field_setter'
+require_relative 'operation_preparers/resolver_type_setter'
+require_relative 'operation_preparers/resource_name_setter'
+
+module GraphqlDevise
+  module MountMethod
+    class OperationPreparer
```

Comment:
Still needs cleanup, just trying to separate behavior for the 3 different operations (default, custom, additional) first